src/CoreServices/FSEvents.cs

```diff
@@ -199,6 +199,30 @@ public FSEventStream (string [] pathsToWatch, TimeSpan latency, FSEventStreamCre
 		{
 		}
 
+		[DllImport (Constants.CoreServicesLibrary)]
+		static extern IntPtr FSEventStreamCreateRelativeToDevice (IntPtr allocator, FSEventStreamCallback callback, 
+		IntPtr context, ulong deviceToWatch, IntPtr pathsToWatchRelativeToDevice, 
+		ulong sinceWhen, double latency, FSEventStreamCreateFlags flags);
+
+		public FSEventStream (CFAllocator allocator, ulong deviceToWatch, NSArray pathsToWatchRelativeToDevice, 
+			ulong sinceWhen, TimeSpan latency, FSEventStreamCreateFlags flags)
+		{
+			if (pathsToWatchRelativeToDevice == null) {
+				throw new ArgumentNullException ("pathsToWatchRelativeToDevice");
```

Comment:
Instead of a string use `name(pathsToWatchRelativeToDevice)` to avoid typos

Reply:
'nameof'